interface/forms/care_plan/new.php

```diff
@@ -146,99 +163,111 @@ function set_related(codetype, code, selector, codedesc) {
-        </td>
-    </tr>
-</table>
-</form>
+		<div class="container">
+        
+			<div class="row">
+				<div class="page-header">
+						<h2><?php echo xlt('Care Plan Form'); ?></h2>
+				</div>
+			</div>
+			<div class="row">
+			<?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/care_plan/save.php?id=" . attr($formid) . "'>\n"; ?>
+				<fieldset>
+					<legend><?php echo xlt('Enter Details'); ?></legend>
+					<?php
+						if (!empty($check_res)) {
+							foreach ($check_res as $key => $obj) {
+					?>
+					<div class = "tb_row" id="tb_row_<?php echo attr($key) + 1; ?>"> 
+						<div class="form-group">
+							<div class=" forms col-xs-3">
+								<label for="code_<?php echo attr($key) + 1; ?>" class="h5"><?php echo xlt('Code'); ?>:</label>
+								<input type="text" id="code_<?php echo attr($key) + 1; ?>"  name="code[]" class="form-control code" value="<?php echo text($obj{"code"}); ?>"  onclick='sel_code(this.parentElement.parentElement.id);'>
+								<span id="displaytext_<?php echo attr($key) + 1; ?>"  class="displaytext help-block"></span>
+								<input type="hidden" id="codetext_<?php echo attr($key) + 1; ?>" name="codetext[]" class="codetext" value="<?php echo text($obj{"codetext"}); ?>">
+							</div>
+							<div class="forms col-xs-4">
+								<label for="description_<?php echo attr($key) + 1; ?>" class="h5"><?php echo xlt('Description'); ?>:</label>
+								<textarea name="description[]"	id="description_<?php echo attr($key) + 1; ?>" class="form-control description"  rows="3" ><?php echo text($obj{"description"}); ?></textarea>
+							</div>
+							<div class="forms col-xs-2">
+								<label for="code_date_<?php echo attr($key) + 1; ?>" class="h5"><?php echo xlt('Date'); ?>:</label>
+								<input type='text' id="code_date_<?php echo attr($key) + 1; ?>" name='code_date[]' class="form-control code_date datepicker" <?php echo attr($disabled) ?> value='<?php echo attr($obj{"date"}); ?>' title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
+							</div>
+							<div class="forms col-xs-2">
+								<label for="care_plan_type_<?php echo attr($key) + 1; ?>" class="h5"><?php echo xlt('Type'); ?>:</label>
+								<select name="care_plan_type[]" id="care_plan_type_<?php echo attr($key) + 1; ?>" class="form-control care_plan_type">
+									<option value=""></option>
+									<?php foreach($care_plan_type as $value):
+									$selected = ($value['value'] == $obj{"care_plan_type"}) ? 'selected="selected"' : '';
+									?>
+									<option value="<?php echo attr($value['value']);?>" <?php echo $selected;?>><?php echo text($value['title']);?></option>
+									<?php endforeach;?>
+								</select>
+							</div>
+							<div class="forms col-xs-1" style="padding-top:35px">
+								<i class="fa fa-plus-circle fa-2x" aria-hidden="true" onclick="duplicateRow(this.parentElement.parentElement.parentElement);" title='<?php echo xla('Click here to duplicate the row'); ?>'></i>
```

Comment:
Suggest using a div with `btn-group` here and removing the `fa-plus-circle` in favor of just a `btn` with non-circle `plus` same for times